response_operations_ui/exceptions/exceptions.py

```diff
@@ -3,3 +3,7 @@ class ApiError(Exception):
     def __init__(self, response):
         self.url = response.url
         self.status_code = response.status_code
+
+
+class NoMessages(Exception):
+    pass
```

Comment:
Good practice is to put a simple docstring here describing when the exception should be raised.